move-mutation-test/src/cli.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright © Eiger
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::too_long_first_doc_paragraph)]
```

Comment:
Is it worth just fixing this? (eg., by adding a newline after the short initial doc)

Reply:
You are right.
Done